saleor/checkout/utils.py

```diff
@@ -695,9 +696,14 @@ def abort_order_data(order_data: dict):
             remove_voucher_usage_by_customer(voucher, order_data["user_email"])
 
 
+def get_order(checkout_token: UUID) -> Optional[Order]:
+    """Get order based on checkout token."""
+    return Order.objects.confirmed().filter(checkout_token=checkout_token).first()
+
+
 @transaction.atomic
 def create_order(
-    *, checkout: Checkout, order_data: dict, user: User, redirect_url: str
+    *, checkout: Checkout, order_data: dict, user: User  # type: ignore
```

Comment:
Why there is `# type: ignore` in function definition?

Reply:
:policewoman:

Reply:
👮‍♀️ 🚔